components/api4ai/api4ai.app.mjs

```diff
@@ -0,0 +1,91 @@
+import { axios } from "@pipedream/platform";
+import FormData from "form-data";
+import fs from "node:fs";
+
+export default {
+  type: "app",
+  app: "api4ai",
+  propDefinitions: {
+    apiKey: {
+      type: "string",
+      label: "API Key",
+      secret: true,
+    },
```

Comment:
Thanks for noticing! We have removed `apiKey` property from "app" and all associated actions.

By the way. In the process of making this change, we relocated the API subscription link (unique to each action) from the `description` of `apiKey` to the `alert` property. However, we are now encountering warnings from `eslint` stating that the `alert` property must have a `label` and `description`. Based on our documentation and previous attempts to push the action's code, it appears that the `alert` property can't have neither a `label` or `description`. This seems to be a linting issue, and it may be necessary to update the global `eslint` configuration.